resources/scripts/ttx_diff.py

```diff
@@ -519,9 +520,60 @@ def remove_gdef_lig_caret_and_var_store(ttx: etree.ElementTree):
         if subtable is not None:
             gdef.remove(subtable)
 
+# remap the elements, and return a map of new_ix -> prev_ix
+def remap_class_def_ids(class_def: etree.ElementTree) -> Dict[int, int]:
```

Comment:
Nit: neither the name nor the commend is informative wrt what this is meant to do. The match sorting in fonttools is informative, perhaps the name could be closer to that? - `sort_class_def_like_fonttools` as it were